<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>530</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>530</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Лабораторная работа 3</string>
  </property>
  <property name="locale">
   <locale language="Russian" country="Russia"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="addLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>280</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>280</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>280</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>Добавление объекта</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="addComboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>135</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>135</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>135</width>
      <height>20</height>
     </size>
    </property>
    <item>
     <property name="text">
      <string>Не выбрано</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Камера</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Куб</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Пирамида</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Компоновщик</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>40</y>
      <width>135</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>135</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>135</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QLabel" name="operationLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>280</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>280</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>280</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>Операции над объектом</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="selectLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>135</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>135</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>135</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>Текущий объект:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="operationComboBox">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>100</y>
      <width>135</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>135</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>135</width>
      <height>20</height>
     </size>
    </property>
    <item>
     <property name="text">
      <string>Не выбрано</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="moveLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>Перенос</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="rotateLabel">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>130</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>Поворот</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="scaleLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>Масштаб.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dXLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>dx</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="oXLabel">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>160</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>ox</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="cXLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>cx</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="dXLineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="oXLineEdit">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>160</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="cXLineEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="cYLineEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>190</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="oYLineEdit">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>190</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="cYLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>190</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>cy</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="dYLineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="oYLabel">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>190</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>oy</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dYLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>dy</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="cZLineEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>220</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="oZLineEdit">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>220</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="cZLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>cz</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="dZLineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="oZLabel">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>220</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>oz</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dZLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>dz</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="moveButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="scaleButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>250</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotateButton">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>250</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>135</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>135</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>135</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>280</y>
      <width>135</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>135</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>135</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>Очистить сцену</string>
    </property>
   </widget>
   <widget class="QLabel" name="compositeLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>280</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>280</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>280</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>Редактирование компоновщика</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="compositeComboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>135</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>135</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>135</width>
      <height>20</height>
     </size>
    </property>
    <item>
     <property name="text">
      <string>Не выбрано</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="objectComboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>135</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>135</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>135</width>
      <height>20</height>
     </size>
    </property>
    <item>
     <property name="text">
      <string>Не выбрано</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="addToCompositeButton">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>340</y>
      <width>135</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>135</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>135</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeFromCompositeButton">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>370</y>
      <width>135</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>135</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>135</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QComboBox" name="selectCameraComboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>135</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>135</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>135</width>
      <height>20</height>
     </size>
    </property>
    <item>
     <property name="text">
      <string>Не выбрано</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="cameraLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>280</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>280</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>280</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>Выбор камеры</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="selectCameraButton">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>430</y>
      <width>135</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>135</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>135</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>Выбрать</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Действия</string>
    </property>
    <addaction name="programAction"/>
    <addaction name="authorAction"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionbebra">
   <property name="text">
    <string>bebra</string>
   </property>
  </action>
  <action name="programAction">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="authorAction">
   <property name="text">
    <string>Об авторе</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
